### ISSUE TITLE
Specify system requirements somewhere

### ISSUE THREAD

**Issue by Zughy** [luanti-org/luanti-org.github.io#300]
I can't find anything about what's the required and recommended system requirements for Minetest. Usecase: I wanted to draw a comparison with Minecraft using official data

**Comment by rubenwardy** [comment]
We don't have required/recommended requirements as we don't have access to enough test devices to determine what they should be.

Also, required and recommended requirements are completely arbitrary, Minecraft could be using a completely different target to us. You're probably better off recording FPS or frametime when viewing similar scenes on the same hardware. If you're able to borrow a chromebook that would probably be quite compelling

**Comment by rollerozxa** [comment]
There used to be a system requirements section on one of the Wiki pages
linked from the downloads page, but it was useless at best and smartass-y
at worst (did you know you need a screen to display graphics? speakers are
optional if you want to hear... sound! Be sure to have a graphics card or
use this thing called `llvmpipe` to render the game :D). The only system
requirements that could be reliably written down would be the OS version
support for each platform and the OpenGL(ES) versions supported. Latter
being OpenGL 1.4 and OpenGLES 2.0 as minimum, but the recommended OpenGL
would be whatever the new opengl3 video driver will use which I assume will
be the baseline sometime in the future.

System requirements for things like CPU, disk space, memory, GPU etc. would
fluctuate wildly depending on your workload and performance pain tolerance.
You can make some absolute minimum requirements like 1GB of RAM (especially
for Android where the OOM killer is very erratic on <1GB RAM devices), or
that you should have a gigabyte or so of spare disk space to allow your
world's map database to grow in singleplayer, but anything beyond that
basically comes down to how high of a view range you want to have at a
target framerate. For e.g. advertising the minimum requirements for an
educator to be able to determine if a school's computer lab or laptop fleet
could handle Minetest, the OpenGL version they have is basically the most
important factor for having it run at all, whether the students can push
60FPS at 200 view range would likely be less important for educational use.

**Comment by Calinou** [comment]
You'll need to define some kind of baseline for comparison, such as a list of specs that can run Minetest in 1920x1080 at a stable 60 FPS on a blank Minetest Game world with no mods. This is what I did for the Godot [System requirements](https://docs.godotengine.org/en/latest/about/system_requirements.html) page (feel free to use it for inspiration).
